pulp_rpm/app/modulemd.py

```diff
@@ -39,7 +40,8 @@ def modules_packages(modules):
         added_modules = current_modules
 
     added_module_packages = modules_packages(added_modules)
-    current_packages = version.content.filter(pulp_type="rpm.package")
+    current_packages = Package.objects \
+        .filter(pk__in=version.content.filter(pulp_type="rpm.package"))
```

Comment:
Both this one and the one on line 32 should instead use ```model_class.get_pulp_type()``` instead of hardcoding the magic string :)

Reply:
Can we do it in a separate PR? Not only this place will be affected, I believe advisories handling is done the same way, so we need to fix this in multiple places.

Reply:
Sure